README.md

```diff
@@ -124,7 +124,6 @@ To uninstall the EOSIO built/installed binaries and dependencies, run:
 1. [White Paper](https://github.com/EOSIO/Documentation/blob/master/TechnicalWhitePaper.md)
 1. [Roadmap](https://github.com/EOSIO/Documentation/blob/master/Roadmap.md)
 
-<a name="gettingstarted"></a>
```

Comment:
How did this even get in here?

Reply:
@dixia  need to create parallel PR to release/2.0.x

Reply:
This does not impact the release branch